app/views/admissions_admin/admissions/statistics.html.haml

```diff
@@ -17,3 +17,27 @@
     %figure
       %figcaption= t('admissions_admin.applications_by_day')
       %img{ src: @applications_per_day_chart }
+
+  %p
+    = "#{@admission.title} hadde #{@known_ids.count} unike søkere, hvorav #{@applicants_who_accepted_role} ble tilbudt"
```

Comment:
Should probably change these to i18n translations using `t(..., ...vars)`

Reply:
Fixed